### ISSUE TITLE
Warning: The `app` argument of `drive_auth_configure()` is deprecated as of googledrive 2.1.0.

### ISSUE THREAD

**Issue by ihecker** [ClaudioZandonella/trackdown#57]
Hi,

I just got the following error when trying to install trackdown from GitHub : 

```
* installing *source* package 'trackdown' ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded from temporary location
Warning: The `app` argument of `drive_auth_configure()` is deprecated as of googledrive
2.1.0.
ℹ Please use the `client` argument instead.
ℹ The deprecated feature was likely used in the trackdown package.
  Please report the issue at
  <https://github.com/claudiozandonella/trackdown/issues>.
Error: package or namespace load failed for 'trackdown':
 .onLoad failed in loadNamespace() for 'trackdown', details:
  call: drive_auth_configure(client = app, path = path, api_key = api_key)
  error: missing(client) || is.null(client) || inherits(client, "gargle_oauth_client") is not TRUE
Error: loading failed
Execution halted
ERROR: loading failed
```

Thank you for your work!

**Comment by ClaudioZandonella** [comment]
Hi @ihecker,

thanks for pointing out this issue!  The new `googledrive` update `2.1.0` changed something in the authentication configure. I will investigate this issue.

Quick hack: installing `googledrive` version `2.0.0` should allow you to install `trackdown`. I know it is not ideal, but I will update to fix the issue

**Comment by ClaudioZandonella** [comment]
Hi @ihecker 

I just released version 1.5.1 that fixed compatibility with the new version of googledrive🎉.

googledirve and gargle (the two packages that manage the communication with Google APIs) changed their underlying functions and arguments. In particular, what was named before app now is client. It is still an ongoing transformation as most functions allow deprecated arguments.

Be sure to install the new version of googledirve and gargle, they are required by trackdown (I already specified it in the Imports section so everything should be automatic).

Now everything should be fixed! I close the issue.

But if there are still problems with the installation, do not hesitate to reopen the issue!😉
